test/user/model.js

```diff
@@ -24,12 +24,12 @@ describe('<Unit Test>', function() {
         });
 
         describe('Method Save', function() {
-            it('should be able to save whithout problems', function(done) {
```

Comment:
I don't fully understand why this test was commented out. It tests an essential feature so I believe your proposed changes should work with the test before merging to ensure we can run the test successfully in the future.